rasa/train.py

```diff
@@ -440,6 +442,53 @@ def get_valid_config(config: Text, mandatory_keys: List[Text]) -> Text:
             "{}. Filling missing parameters from fallback configuration file: '{}'."
             "".format(config, ", ".join(missing_keys), FALLBACK_CONFIG_PATH)
         )
-        enrich_config(config_path, missing_keys, FALLBACK_CONFIG_PATH)
+        _enrich_config(config_path, missing_keys, FALLBACK_CONFIG_PATH)
 
     return config_path
+
+
+def _package_model(
```

Comment:
should we move this to `model.py` and maybe write tests?

Reply:
I would like to keep the function in `train.py` as it has a `print_sucess` statement and its functionality is only used in this file. But will write some tests.